### DISCUSSION TITLE
Android : Upload certain folder even when .nomedia is there

### DISCUSSION THREAD

**Discussion by ghost** [ente-io/ente#959] [Enhancements]
Hello I have been using ente for some time with a Samsung phone and I use folders to separate work and personal photos and I want ente to sync a folder containing personal photos but it doesn't because of the .nomedia folder which I have placed to avoid them from showing up on the Samsung gallery app 

Is it possible for ente to upload images from a folder that contains the .nomedia file as well

**Comment by ua741** [comment]
Hello @lalisawhiteheart,

Thank you for choosing ente.

This seems like an advanced use-case that we don't support yet (more details below).

For uploading them, you can always select those photos and share[1] them with `ente`. The app will upload those photos in the background. If a photo is already uploaded, it won't be uploaded again. If your original photo had location metadata, you may want to verify that the same is present in the uploaded photo (Open file -> Info -> Exif).

I understand that it's not an ideal solution, but I'm afraid that's our only option right now.

---
ente only requests media permission. From our understanding, because of the `.nomedia` flag, the OS doesn't treat these files as media files.
To support this use-case, we need to build support to access all types of files from a particular location on the device.

[1] 
<img src="https://github.com/ente-io/photos-app/assets/254676/eeada1c7-ab4f-4623-9a10-ba3a375d6cb8" width="400">

**Comment by ghost** [comment]
I would like this feature to be implemented in the future if you guys think its possible and regarding my issue I will look for a work around if any exists

**Comment by ghost** [comment]
I would also appreciate this whether it's implemented using the .nomedia file or other means. At the very least, I want to have an option to ignore folders permanently instead of seeing them prominently featured on top as "on device". Google Photos has this feature and I cannot imagine that it requires rocket science.
